### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/explore/explore-interactive.html

### PR DESCRIPTION
**Pull request by anubha-v-ardhan**
This PR is part of ongoing `hi` localization effort here #29353 

/langauge hi

### CODE REVIEW

**Comment by mittalyashu** [comment]
Haven’t I already reviewed this file in some other PR? 🤔

**Comment by anubha-v-ardhan** [comment]
Can you point out the PR. The content of many files are similar(but not same), especially in `content/en/docs/tutorials/kubernetes-basics` directory. It is very much possible you reviewed a similar PR.
To avoid conflicts, we add the PR under the filename on localization issue's checklist with the id of person who worked / is working on it.

File: content/hi/docs/tutorials/kubernetes-basics/explore/explore-interactive.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: इंटरैक्टिव ट्यूटोरियल - अपने ऐप को एक्सप्लोर करना
```
Comment (divya-mohan0209) [inline content/hi/docs/tutorials/kubernetes-basics/explore/explore-interactive.html:2]: Would अन्वेषण करना be better here for consistency?

**Comment by anubha-v-ardhan** [comment]
/approve
ref. https://github.com/kubernetes/website/pull/29403#issuecomment-995780451

**Pull request merged**